### DISCUSSION TITLE
Failed to Install unstructured[pdf] in python:3.11-alpine docker image

### DISCUSSION THREAD

**Discussion by hteeyeoh** [Unstructured-IO/unstructured#2154] [Q&A]
Hi all, I tried to install unstructured[pdf] in python:3.11-alpine docker image for our GEN-AI related langchain operation. Throughout the process, I also installed those dependencies apk where internally unstructured[pdf] tried to install opencv-python and due to pip backtracking runtime it will "looking at multiple versions of setuptools to determine which version is compatible with other requirements." and will hit into some version conflict issue of numpy as show in below snippet log"

![image](https://github.com/Unstructured-IO/unstructured/assets/57510309/c2c0edce-4916-469c-b33e-16669f415e37)

Attached the complete build log as well here
[alpine_numpy_error_build_log.txt](https://github.com/Unstructured-IO/unstructured/files/13456939/alpine_numpy_error_build_log.txt)

**Comment by kauabh** [comment]
Facing similar issue,  anyone able to resolve this?

**Reply by hteeyeoh** [reply]
using another base image solved this issue. instead of using python3.11-alpine, use python3.10-alpine
